test/integration_test.rb

```diff
@@ -160,9 +172,9 @@ def sign_params(params)
     params
   end
 
-  def expect_access_token_request(access_token)
+  def expect_access_token_request(access_token, scope)
     FakeWeb.register_uri(:post, "https://snowdevil.myshopify.com/admin/oauth/access_token",
-                         body: JSON.dump(access_token: access_token),
+                         body: JSON.dump(access_token: access_token, scope: scope),
```

Comment:
It will once I merge the pending PR in Shopify (you're pinged on it).

I will take care of the docs.